src/actors/_actorsheets.scss

```diff
@@ -190,6 +190,13 @@
       font-weight: bold;
     }
 
+    input[name="system.extendedName"] {
+      background-color: transparent;
```

Comment:
What do you think about adding
`color: var(--color-text-dark-secondary);`
to give more emphasis to the actual name - it feels the two name headlines are competing abit for the attention otherwise.

![image](https://github.com/sun-dragon-cult/fvtt-system-rqg/assets/4853677/502c399e-9322-4325-a08d-31215d25b132)